### PR TITLE
add(epp): applications-sailpoint-identitynow-restapi

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [X] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+| sources.status.unchecked.count |       |
+| source health status           |       |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+
+Pour la supervision, un utilisateur avec des droits de lecture sur l'[API](https://developer.sailpoint.com/idn/api/getting-started) SailPoint IdentityNow est nécessaire.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:81]:
```suggestion
Pour la supervision, un utilisateur avec des droits de lecture sur l'[API SailPoint IdentityNow](https://developer.sailpoint.com/idn/api/getting-started) est nécessaire.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+2. Renseignez les macros désirées (par exemple, ajustez les seuils d'alerte). Les macros indiquées ci-dessous comme requises (**Obligatoire**) doivent être renseignées.
+
+<Tabs groupId="sync">
+<TabItem value="Search-Count" label="Search-Count">
+
+| Macro                | Description                                                                                         | Valeur par défaut           | Obligatoire |
+|:---------------------|:----------------------------------------------------------------------------------------------------|:----------------------------|:------------|
+| OUTPUT               | Output to print after retrieving the count of results (Default: "Number of results: %{count}")      | Number of results: %{count} |             |
+| QUERY                | Query parameters used to construct an Elasticsearch query object(see documentation)                 |                             |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:201]:
```suggestion
| QUERY                | Query parameters used to construct an Elasticsearch query object (see documentation)                 |                             |             |
```

Comment (cg-tw) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:201]: was there going to be a link on "documentation"?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+| --proxyurl                                 | Proxy URL. Eg: http://my.proxy:3128                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Http global                |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:336]:
```suggestion
| --proxyurl                                 | Proxy URL. E.g.: `http://my.proxy:3128`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Http global                |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+| --statefile-cipher                         | Define the cipher algorithm to encrypt the cache (Default: 'AES').                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Retention                  |
+
+#### Options des modes
+
+Les options spécifiques aux modes sont listées ci-dessus :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:357]:
```suggestion
Les options spécifiques aux modes sont listées ci-dessous :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+```
+
+### Diagnostic des erreurs communes
+
+Rendez-vous sur la [documentation dédiée](../getting-started/how-to-guides/troubleshooting-plugins.md#http-and-api-checks)
+des plugins basés sur HTTP/API.
+
+### Modes disponibles
+
+Tous les modes disponibles peuvent être affichés en ajoutant le paramètre
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:272]:
```suggestion
Dans la plupart des cas, un mode correspond à un modèle de service. Le mode est renseigné dans la commande d'exécution 
du connecteur. Dans l'interface de Centreon, il n'est pas nécessaire de les spécifier explicitement, leur utilisation est
implicite dès lors que vous utilisez un modèle de service. En revanche, vous devrez spécifier le mode correspondant à ce
modèle si vous voulez tester la commande d'exécution du connecteur dans votre terminal.
Tous les modes disponibles peuvent être affichés en ajoutant le paramètre
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md [hunk 6]
```diff
@@ -0,0 +1,391 @@
...
+|:--------------|:------------------------------------------------|
+| list-sources  | Used for service discovery                      |
+| search-count  | App-Sailpoint-Identitynow-Restapi-Search-Count  |
+| source-health | App-Sailpoint-Identitynow-Restapi-Source-Health |
+
+### Options disponibles
+
+#### Options génériques
+
+Les options génériques aux modes sont listées ci-dessous :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:293]:
```suggestion
Les options génériques sont listées ci-dessous :
```

File: pp/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md [hunk 7]
```diff
@@ -0,0 +1,389 @@
...
+
+* **App-Sailpoint-Identitynow-Restapi-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="App-Sailpoint-Identitynow-Restapi-custom" label="App-Sailpoint-Identitynow-Restapi-custom">
+
+| Service Alias | Service Template  | Service Description |
+|:--------------|:------------------|:--------------------|
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:22]: bug?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md [hunk 8]
```diff
@@ -0,0 +1,391 @@
...
+* **App-Sailpoint-Identitynow-Restapi-custom**
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="App-Sailpoint-Identitynow-Restapi-custom" label="App-Sailpoint-Identitynow-Restapi-custom">
+
+| Alias | Modèle de service | Description |
+|:------|:------------------|:------------|
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:23]: bug?

File: pp/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md [hunk 9]
```diff
@@ -0,0 +1,389 @@
...
+
+> The services listed above are created automatically when the **App-Sailpoint-Identitynow-Restapi-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias | Service Template                                | Service Description                                     | Discovery  |
+|:--------------|:------------------------------------------------|:--------------------------------------------------------|:-----------|
+| Search-Count  | App-Sailpoint-Identitynow-Restapi-Search-Count  | Performs a search with a provided query and returns the |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:31]: part of the sentence is missing

File: pp/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md [hunk 10]
```diff
@@ -0,0 +1,389 @@
...
+| --statefile-cipher                         | Define the cipher algorithm to encrypt the cache (Default: 'AES').                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Retention                  |
+
+#### Modes options
+
+All modes specific options are listed here:
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:355]:
```suggestion
All mode-specific options are listed here:
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md (hunk 7 above)
Comment (garnier-quentin) [reply pp/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:22]: there is no service attached by default (normal)

File: pp/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md [hunk 11]
```diff
@@ -0,0 +1,389 @@
...
+</TabItem>
+</Tabs>
+
+3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of service, and on page **Resources Status**. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
+
+## How to check in the CLI that the configuration is OK and what are the main options for?
+
+Once the plugin is installed, log into your Centreon poller's CLI using the
+**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
+is able to monitor a server using a command like this one (replace the sample values by yours):
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md:235]:
```suggestion
is able to monitor the resource using a command like this one (replace the sample values by yours):
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**